cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1555,6 +1594,13 @@ public void onFailure(int statusCode, String response, Throwable throwable) {
         })).start());
     }
 
+    public void removeSubscriptionTags(String[] tagIds) {
```

Comment:
Same like `add` methods. Please handle the logic here rather than in the existing method.